src/handlers/GraphQLHandler.ts

```diff
@@ -176,9 +176,10 @@ Consider naming this operation or using "graphql.operation" request handler to i
     const hasMatchingOperationType =
       this.info.operationType === 'all' ||
       parsedResult.operationType === this.info.operationType
+
     const hasMatchingOperationName =
       this.info.operationName instanceof RegExp
-        ? this.info.operationName.test(parsedResult.operationName)
+        ? this.info.operationName.test(parsedResult.operationName || '')
```

Comment:
At this point in time `parsedResult.operationName` cannot be undefined, null, or an empty string. 
Was it some TypeScript error that made you add this fallback (`|| ''`)?